docs/data/toolpad/core/components/account/AccountSlotsWallet.js

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import {
+  Avatar,
+  Button,
+  Divider,
+  Typography,
+  Stack,
+  IconButton,
+} from '@mui/material';
+import WalletIcon from '@mui/icons-material/AccountBalance';
+import SendIcon from '@mui/icons-material/Send';
+import ShoppingCart from '@mui/icons-material/ShoppingCart';
+import CopyIcon from '@mui/icons-material/ContentCopy';
+
+import {
+  Account,
+  AccountPreview,
+  AuthenticationContext,
+  SessionContext,
+  SignOutButton,
+} from '@toolpad/core';
+import { AccountPopoverFooter } from '../../../../../../packages/toolpad-core/src/Account/AccountPopoverFooter';
```

Comment:
Really long import, shouldn't this component be importable from `@toolpad/core/Account` probably?